web/src/components/storage/dasd/DASDPage.tsx

```diff
@@ -20,9 +20,8 @@
  */
 
 import React from "react";
-import DASDTable from "~/components/storage/DASDTable";
+import { DASDTable, DASDFormatProgress } from "~/components/storage/dasd";
```

Comment:
These components are in the same scope that the one importing them. Please, use relative route and do not export them if not used outside.